### ISSUE TITLE
Improve Alt Text for Images

### ISSUE THREAD

**Issue by aziznab** [aziznab/uh-accessibility-2024#1]
Ensure all images have descriptive alt text to enhance accessibility.

**Comment by aziznab** [comment]
![image](https://github.com/user-attachments/assets/e5a2362d-dc97-4220-85bf-35b6ff882264)
![image](https://github.com/user-attachments/assets/2eee720b-a0e3-45ee-b97b-4b1959b18cc7)
![image](https://github.com/user-attachments/assets/fc70d8d5-940e-487c-86b0-05770e7f9be1)

| #  | Severity (0-4) | Heuristic Transgressed/Supported | Where in the interface/application | Issue                                           | Remedy                                             |
|----|----------------|----------------------------------|------------------------------------|-------------------------------------------------|----------------------------------------------------|
| 1  | 2              | Match between system and real world | About You section                  | Alternative text is insufficient or contains extraneous information | Update alt text to be more descriptive and relevant |
